functions.rb

```diff
@@ -0,0 +1,92 @@
+class Function
+  def initialize
+    @books = []
+    @rental_data = []
+    @people = []
+  end
+end
+
+def show_books(books)
+  if @books.empty?
+    puts 'No books found in library'
+  else
+    books.each do |book|
+      puts "title: #{book.title}, Author: #{book.author}"
+    end
+  end
+end
```

Comment:
- [ OPTIONAL ] I highly recommend to fix the method here since when I choose list all books I can't see any books listed in the terminal so please work on it to fix 💯🔥✅♻😊

![image](https://user-images.githubusercontent.com/58553711/214845415-310ea3af-6d12-46c9-9f0f-faac82e9e1e6.png)